plugins/sep/test/sepsource-tests.cc

```diff
@@ -294,7 +296,7 @@ BOOST_AUTO_TEST_CASE(sepsource_fill_sli_1) {
   sepSource->fillSliLayerBitmap(sli);
   BOOST_CHECK(sli->width == 600);
   BOOST_CHECK(sli->height == 400);
-  BOOST_CHECK(sli->spp == 4);
+  //BOOST_CHECK(sli->spp == 4); No longer specifically 4
```

Comment:
If the input file has 4 channels (and this one does), then it should be 4, right?

Reply:
Fixed